### PR TITLE
Move ONNX Transformer into Microsoft.ML.Transforms.Onnx namespace.

### PR DESCRIPTION
**Pull request by codemzs**
towards #2751

### CODE REVIEW

File: src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNetExtension.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.Runtime;
 
-namespace Microsoft.ML.Transforms
+namespace Microsoft.ML.Transforms.Onnx
```
Comment (sfilipi) [inline src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNetExtension.cs]: all other extensions seem to be in the Microsoft.ML namespace. Let's leave this and the Resnet's there too.

Comment (sfilipi) [reply src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNetExtension.cs]: Signing off, but please move the extensions.

Comment (codemzs) [reply src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNetExtension.cs]: thanks, done.

File: src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs [hunk 2]
```diff
@@ -6,8 +6,10 @@
 using System.IO;
 using Microsoft.ML.Data;
 using Microsoft.ML.Runtime;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.Onnx;
 
-namespace Microsoft.ML.Transforms
+namespace Microsoft.ML
```
Comment (TomFinley) [inline src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs]: Ah, nice, I had forgotten about these extensions.

**Pull request merged**